### PR TITLE
Move atom plotting function to horton.scripts.atomdb

### LINKED ISSUE #45: Put the atom plotting function in horton-atomdb.py in the HORTON library
This makes it possible to plot proatom databases from different sources, which is convenient. I have a branch with the code. Needs to be rebased and cleaned up. (Also, this function should become a method of ProAtomDB.)

### PR DESCRIPTION
**Pull request by tovrstra**
Fixes #45

### CODE REVIEW

**Comment by tovrstra** [comment]
Ready for review.

File: horton/scripts/test/test_atomdb.py [hunk 1]
```diff
@@ -196,3 +196,14 @@ def test_script_convert_g03():
         assert padb.get_numbers() == [1, 8]
         assert padb.get_charges(1) == [0, -1]
         assert padb.get_charges(8) == [+1, 0, -1]
+
+
+def test_plot_atoms():
+    padb = ProAtomDB.from_refatoms(numbers=[8, 1], max_kation=1, max_anion=1)
```
Comment (matt-chan) [inline horton/scripts/test/test_atomdb.py]: Cation is with a C. Was the K intentional?

Comment (tovrstra) [inline horton/scripts/test/test_atomdb.py]: Oops! That is one of my favorite spelling errors: a horrible Dutchism embedded in my muscle memory. Are you OK with fixing this throughout the code? I would do that in a separate PR.

**Comment by matt-chan** [comment]
Yep I'll fix it up!

**Comment by matt-chan** [comment]
Looks good. Merging
